doc/Doxyfile

```diff
@@ -217,7 +217,7 @@ JAVADOC_BANNER         = NO
 # requiring an explicit \brief command for a brief description.)
 # The default value is: NO.
```

Comment:
Needs update since you changed the default value

Reply:
That's the default value for the upstream example Doxyfle, so we'll keep the text as is here.